src/App.tsx

```diff
@@ -8,11 +10,31 @@ export const tabs: Tab[] = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
+  const onTabSelected = (tab: Tab) => {
+    if (selectedTab.id !== tab.id) {
+      setSelectedTab(tab);
+    }
+  };
+
+  return (
+    <div className="content is-medium">
+      <h1>
+        Selected tab is
+        {' '}
+        {selectedTab.title}
+      </h1>
+      <Tabs
+        tabs={tabs}
+        selectedTabId={selectedTab.id}
+        onTabSelected={onTabSelected}
+      />
+
+      <p data-cy="tab-content">{selectedTab.content}</p>
```

Comment:
- Implement the Tabs component accepting tabs as a prop and displaying a link per each tab and 
**the content of the selected tab**
So move your content rendering to `Tabs` component